### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by pkalever**
### What does this PR achieve? Why do we need it?

* create: defend on the use of hostnames                                 
                                                                       
We have noticed that our users are trying to create block volume with  
hostnames and reporting issues, so it would be better if we defend it. 
                                                                       
```
$ gluster-block create sample-test/block block.lab.green.com 1MiB      
hostnames are not supported with gluster-block                         
Hint: please use ip addresses only                                     
                                                                       
$ gluster-block create sample-test/block 192.168.43.158 1MiB           
IQN: iqn.2016-12.org.gluster-block:2d4a2dfb-db5d-401c-b21a-01c57ad38705
PORTAL(S):  192.168.43.158:3260                                        
RESULT: SUCCESS                                                        

```                                                                       

* MSG: do not override last character from the msg                 
                                                                 
```
$ gluster-block create sample-test/block block.lab.green.com 1MiB
hostnames are not supported with gluster-bloc                    
Hint: please use ip addresses onl                                

```

Signed-off-by: Prasanna Kumar Kalever \<prasanna.kalever@redhat.com\>

### CODE REVIEW

File: cli/gluster-block.c [hunk 1]
```diff
@@ -566,6 +582,7 @@ glusterBlockCreate(int argcount, char **options, int json)
   blockCreateCli cobj = {{0}, };
   bool TAKE_SIZE=true;
   bool PREALLOC_OPT=false;
+  int i;
```
Comment (lxbsz) [inline cli/gluster-block.c]: Please fix:

gluster-block.c: In function 'glusterBlockCreate':
gluster-block.c:585:7: warning: unused variable 'i' [-Wunused-variable]
   int i;
       ^

Comment (pkalever) [reply cli/gluster-block.c]: @lxbsz sure, thanks!

**Comment by pkalever** [comment]
@lxbsz done now, please take a look. Thanks!

**Comment by pkalever** [comment]
@amarts @lxbsz Thanks, merging now.

**Pull request merged**